### PR TITLE
feat($cli): run debug mode without clearing screen (close #2100)

### LINKED ISSUE #2100: --debug goes with --no-clear-screen automatically in dev mode
#### What problem does this feature solve?

Anyone who run `vuepress dev --debug` want to see the logs. But if they don't run with `--no-clear-screen`, logs will be cleaned.

#### What does the proposed API look like?

`--debug` goes with --no-clear-screen automatically in dev mode.

#### Are you willing to work on this yourself?
Yes.

### PR DESCRIPTION
**Pull request by newsbielt703**
**Summary**

please head #2100.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/zh/api/cli.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ vuepress <command> targetDir [options]
 当服务端准备就绪时自动打开浏览器。
 
 ### --no-clear-screen
-当 dev server 就绪时不清除屏幕。
+当 dev server 就绪时不清除屏幕。请注意dev server不會在调试模式下清除屏幕。
```
Comment (ludanxer) [inline packages/docs/docs/zh/api/cli.md]:
```suggestion
当 dev server 就绪时不清除屏幕。请注意 dev server 不会在调试模式下清除屏幕。
```
Add spaces between English and Chinese, and use Simplified Chinese

Comment (newsbielt703) [reply packages/docs/docs/zh/api/cli.md]: Thanks for catching.

**Comment by meteorlxy** [comment]
I found a strange behavior of cac:

If users pass `--no-clear-screen`, cac will get:

```js
{
  'clear-screen': true,
  clearScreen: false,
}
```

If users do not pass `--no-clear-screen`, cac will get:

```js
'clear-screen': true,
```

File: packages/@vuepress/core/lib/node/dev/index.js [hunk 2]
```diff
@@ -175,7 +175,7 @@ module.exports = class DevProcess extends EventEmitter {
         port: this.port,
         displayHost: this.displayHost,
         publicPath: this.context.base,
-        clearScreen: this.context.options.clearScreen
+        clearScreen: this.context.options.clearScreen || !this.context.options.debug
```
Comment (meteorlxy) [inline packages/@vuepress/core/lib/node/dev/index.js]: Let's upgrade cac to 6.5.4 https://github.com/cacjs/cac/issues/61

**Pull request merged**